src/streams.h

```diff
@@ -626,6 +626,11 @@ class CAutoFile
         }
     }
 
+    void write(const std::string& s)
+    {
+        write(MakeByteSpan(s));
+    }
```

Comment:
Probably better to do this inside `CreateUTXOSnapshot`

Reply:
eg https://github.com/luke-jr/bitcoin/commit/5ddb75caa0ab2e3be306b96132e3b9af7ba62a40 (but since you don't have all the params in this PR, it will take a bit of rebasing)

Reply:
Done. Thanks.